w09h01/test/pgdp/pingutrip/kidFriedlyTripTest.java

```diff
@@ -0,0 +1,45 @@
+package pgdp.pingutrip;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class kidFriedlyTripTest {
+    @Test
+    @DisplayName("Checks if really stops in kidFriendlyTripTest when the first element is longer then the average ")
+    void kidFriendlyTripTest(){
+        List<OneWay> lists = List.of(new OneWay(new WayPoint(0.0, 0.0), new WayPoint(4.0, 0.0)),
+                new OneWay(new WayPoint(0.0, 0.0), new WayPoint(1.0, 0.0)),
+                new OneWay(new WayPoint(1.0, 0.0), new WayPoint(3.0, 0.0)));
+        List<OneWay> result = List.of();
+        assertArrayEquals(PinguTrip.kidFriendlyTrip(lists), result);
+    }
+
+    @Test
+    @DisplayName("Checks if every element gets returned, checks for the correctness of the math")
+    void kidFriendlyTripTest2(){
+        List<OneWay> lists = List.of(new OneWay(new WayPoint(2.0, 0.0), new WayPoint(4.0, 0.0)),
+                new OneWay(new WayPoint(0.0, 0.0), new WayPoint(2.0, 0.0)),
+                new OneWay(new WayPoint(0.0, 0.0), new WayPoint(2.0, 0.0)));
+        List<OneWay> result = List.of(new OneWay(new WayPoint(2.0, 0.0), new WayPoint(4.0, 0.0)),
+                new OneWay(new WayPoint(0.0, 0.0), new WayPoint(2.0, 0.0)),
+                new OneWay(new WayPoint(0.0, 0.0), new WayPoint(2.0, 0.0)));
+        assertArrayEquals(PinguTrip.kidFriendlyTrip(lists), result);
+    }
```

Comment:
```suggestion
    void kidFriendlyTripTest2() {
        List<OneWay> lists = List.of(new OneWay(new WayPoint(2.0, 0.0), new WayPoint(4.0, 0.0)),
                new OneWay(new WayPoint(0.0, 0.0), new WayPoint(2.0, 0.0)),
                new OneWay(new WayPoint(0.0, 0.0), new WayPoint(2.0, 0.0)));

        List<OneWay> expected = List.of(new OneWay(new WayPoint(2.0, 0.0), new WayPoint(4.0, 0.0)),
                new OneWay(new WayPoint(0.0, 0.0), new WayPoint(2.0, 0.0)),
                new OneWay(new WayPoint(0.0, 0.0), new WayPoint(2.0, 0.0)));
        List<OneWay> actual = PinguTrip.kidFriendlyTrip(lists);

        assertArrayEquals(expected.toArray(), actual.toArray());
    }
```

This is more readable 👍